src/autogluon/bench/datasets/multimodal_dataset.py

```diff
@@ -36,13 +37,36 @@
     "NewsChannel",
 ]
 
+logger = logging.getLogger(__name__)
+
 
 def _path_expander(path, base_folder):
     path_l = path.split(";")
     return ";".join([os.path.abspath(os.path.join(base_folder, path)) for path in path_l])
 
 
 class BaseMultiModalDataset(abc.ABC):
+    def __init__(self, split: str, dataset_name: str, data_info: dict):
+        """
+        Initializes the class.
+
+        Args:
+            split (str): Specifies the dataset split. It should be one of the following options: 'train', 'val', 'test'.
+        """
+        try:
+            ext = data_info[split]["url"].split(".")[-1]
```

Comment:
probably better to use `https://docs.python.org/3/library/os.path.html#os.path.splitext`

Reply:
Neat! I didn't know this.